README.md

```diff
@@ -1 +1,26 @@
 # FINDOLOGIC Shopware 6 Plugin Override
+
+This is a basic plugin skeleton which may be used to customize
+[FINDOLOGIC's Shopware 6 plugin](#).
+
+## Requirements
+Note that in order to run this plugin you will need to install
+[FINDOLOGIC Search](#) first.
+
+## Installation
+The procedure is basically the same as for any other Shopware plugin. It can be
+summed up in a few simple steps:
+* Activate the plugin under 
+    - Settings > System > Plugins 
+* Open the Config context menu and 
+    - set the field "Active" to "Yes"
+    - set the "shopkey" to a sales channel
+* Click "Save"
+* Finally, the shop's cache must be cleared
+
+## License
```

Comment:
Having a link to the license is not really necessary. I would remove it, but keep it if you want.